src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ActionBar.xaml.cs

```diff
@@ -36,4 +43,6 @@ private void ActionListViewItem_Tapped(object sender, TappedRoutedEventArgs e)
             ViewModel?.InvokeItemCommand.Execute(item);
         }
     }
+
+    public void Receive(UpdateActionBarPage message) => ViewModel.CurrentPage = message.Page;
```

Comment:
I feel like this is a bad merge, we intentionally had moved this back to the ViewModel as it was just updating the page in the other PR, right?

Reply:
Oof yea, during experiments I reverted this back so I could have the view do the PropertyChanged thing